MobileWallet/Screens/Debug/UIViewControllerDebugExtension.swift

```diff
@@ -217,12 +217,44 @@ extension UIViewController: MFMailComposeViewControllerDelegate {
                 })
             }
         } else {
-            UserFeedback.shared.error(title: "Feedback failed", description: "Apple mail app needs to be setup to be able to send bug report mails")
+            shareFeedback()
         }
 
         TariLogger.info("Feedback shared")
     }
 
+    private func shareFeedback() {
+        do {
+            let message = "Tari iOS bug report \(TariSettings.shared.bugReportEmail)"
+            let archiveURL = try zipDebugFiles()
```

Comment:
The zip file would have just been created, this would create a second one. Maybe refactor so just one zip is created when I user taps the "Report a bug" button.